pkg/annotations/annotations.go

```diff
@@ -4,6 +4,9 @@ const (
 	// UsernsMode is the user namespace mode to use
 	UsernsModeAnnotation = "io.kubernetes.cri-o.userns-mode"
 
+	// UnifiedCgroupAnnotation is the user namespace mode to use
```

Comment:
this comment seems incorrect :)

Reply:
thanks for noticing my copy&pasta.  I'll fix it